src/utils/save-token-to-ls.ts

```diff
@@ -1,8 +1,8 @@
-import { LSKeys } from '../sources/enums/ls-keys';
+import type { LSKeys } from '../sources/enums/ls-keys';
 import type { Auth } from '../sources/types/auth';
 
-export const saveTokenToLS = (token: Auth.Token) => {
+export const saveTokenToLS = (token: Auth.Token, key: LSKeys) => {
```

Comment:
коммент просто чтоб доколупаться, править не обязательно))

у тебя в лс значение записывается как localStorage.setItem(key, value);
и поэтому просится аргументы в функцию принимать в таком жэе порядке) это как бы не важно, просто мозгу запинается, когда такой порядок видит


```suggestion
export const saveTokenToLS = (key: LSKeys, token: Auth.Token) => {
```